lib/live_fixtures/import/fixtures.rb

```diff
@@ -77,6 +89,33 @@ def model_connection
 
     private
 
+    # Uses the underlying map of labels to return the referenced model's newly
+    # assigned ID.
+    # @raise [LiveFixtures::MissingReferenceError] if the label isn't found.
+    # @param label_to_fetch [String] the label of the referenced model.
+    # @return [Integer] the newly assigned ID of the referenced model.
+    def fetch_id_for_label(label_to_fetch)
+      @label_to_id.fetch(label_to_fetch)
+    rescue KeyError
+      raise LiveFixtures::MissingReferenceError, <<-ERROR.squish
+      Unable to find ID for model referenced by label #{label_to_fetch} while
+      importing #{model_class} from #{table_name}.yml. Perhaps it isn't included
+      in these fixtures or it is too late in the insert_order and has not yet
+      been imported.
+      ERROR
+    end
+
+    # Checks if the string looks like a label referencing the specified table.
+    # @param label_to_check [String] the string that might be a label.
+    # @param table_name [String] the table the label might reference.
+    # @return [Boolean] whether the string is a label for the table.
+    # @see LiveFixtures::Export::Reference
+    # @see LiveFixtures::Export::Fixture#yml_value
+    def is_label_for_table?(label_to_check, table_name)
+      label_prefix = table_name + "_"
+      label_to_check.starts_with? label_prefix
+    end
```

Comment:
Can you help me understand why we are doing this? I don't know that we need to assume labels are in a particular format.